commands/dns-records.go

```diff
@@ -17,12 +19,61 @@ type DNSRecords struct {
 	BaseCommand
 }
 
+// DNSRecord is the struct for a single DNS entry
+type DNSRecord struct {
+	Id      string // nolint
+	Name    string
+	Image   string
+	IPs     []string
+	TTL     int64
+	Aliases []string
+}
+
+// DNSRecordsList is an array of DNSRecords
+type DNSRecordsList []*DNSRecord
+
+const (
+	unixHostsPreamble  = "##+++ added by rig"
```

Comment:
I think you'll use this regardless of the underlying os as markers for starting/removing items so perhaps just call this hostsPreamble

Reply:
Good point.  Thanks.